src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -1058,6 +1058,29 @@ components:
           description: |
             An ISO-8601 date representing date that the Review is due.
           example: '2023-11-19'
+        completedDate:
```

Comment:
I cant remember the UI for reviewing a plan ... does it ask a date when the review was done? IE. a different date to the automatically collected createdAt/updatedAt fields? 🤔